### PR TITLE
Some tweaks in wording and small fixes

### PR DESCRIPTION
**Pull request by DimitrisJim**

### CODE REVIEW

**Comment by willingc** [comment]
Looks good @DimitrisJim. Thanks for the attention to detail.

File: gitbootcamp.rst [hunk 1]
```diff
@@ -77,13 +77,15 @@ To switch to a different branch::
...
--------------------
+Deleting Local Branches
+-----------------------
 
-To delete branch that you no longer need::
+To delete a branch that you no longer need::
 
    $ git branch -D <branch-name>
 
+You may specify more than one branches for deletion.
```
Comment (brettcannon) [inline gitbootcamp.rst]: "more than one branch"

Comment (willingc) [inline gitbootcamp.rst]: Good catch @brettcannon

**Comment by DimitrisJim** [comment]
Silly mistake :smile:. Fixed that up, thanks!

**Comment by brettcannon** [comment]
Thanks!

**Pull request merged**